nupic/research/frameworks/vernon/distributed/experiments/components/distributed_base.py

```diff
@@ -19,17 +19,20 @@
 # http://numenta.org/licenses/
 # ----------------------------------------------------------------------
 
-import copy
-
 import torch.distributed as dist
 from torch import multiprocessing
 
+from nupic.research.frameworks.vernon import interfaces
+
 __all__ = [
-    "Distributed",
+    "DistributedBase",
 ]
 
 
-class Distributed:
+class DistributedBase(
+    interfaces.DistributedAggregation,  # Implements
```

Comment:
You explained to us the distinction between Implements and Requires "tags" and why do we use them, but maybe we can add a line about it in the documentation as well, if we haven't already (that first README can be a good place). 

It might sound obvious after some thinking, but I don't think it is at first glance to someone who is skimming through the library to learn how it works.

Reply:
Definitely agree, I was in the process of writing that.